src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -161,14 +161,34 @@ trait MarathonConf extends ScallopConf with ZookeeperConf with IterativeOfferMat
     noshort = true
   )
 
-  //Internal settings, that are not intended for external use
-  lazy val internalStoreBackend = opt[String]("internal_store_backend",
-    descr = "The backend storage system to use. One of zk, mesos_zk, mem",
+  lazy val storeBackend = opt[String]("store_backend",
```

Comment:
Nobody should know that parameter. Do you think this is a problem?
I would like to reserve `--internal*` for experimental flags.